schemas/rum/_common-schema.json

```
@@ -204,6 +204,70 @@
       },
       "readOnly": true
     },
+    "os": {
+      "type": "object",
+      "description": "Operating system properties",
+      "required": [
+        "name",
+        "version",
+        "version_major"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Operating system name, ex. Android, iOS",
```

nit: I believe it's [not the right abbreviation](https://www.quora.com/Is-example-abbreviated-as-i-e-ex-or-e-g) but I might be wrong 🙃
```suggestion
          "description": "Operating system name, e.g. Android, iOS",
```

---

yes, you are right